src/components/Filter.js

```diff
@@ -0,0 +1,75 @@
+// import * as json from "src/mock-data/tickets.json";
```

Comment:
Чтобы импортировать json файл, можно написать 
```suggestion
import ticketsJson from '.../tickets.json'
```